### PR TITLE
Clean up Feeder task role

### PR DESCRIPTION
**Pull request by farski**

### CODE REVIEW

File: stacks/apps/feeder.yml [hunk 1]
```diff
@@ -251,47 +251,49 @@ Resources:
       Policies:
         - PolicyDocument:
             Statement:
-              - Action: sqs:*
+              - Action:
+                  - sqs:ChangeMessageVisibility
+                  - sqs:Get*
+                  - sqs:List*
+                  - sqs:ReceiveMessage
+                  - sqs:SendMessage
```
Comment (cavis) [inline stacks/apps/feeder.yml:259]: Did this list come from Shoryuken?

File: stacks/apps/feeder.yml [hunk 2]
```diff
@@ -251,47 +251,49 @@ Resources:
...
                 Effect: Allow
                 Resource:
-                  - !Sub arn:${AWS::Partition}:sqs:${AWS::Region}:${AWS::AccountId}:*_feeder_*
                   - !GetAtt DefaultJobQueue.Arn
-                  - !GetAtt DefaultJobDeadletterQueue.Arn
                   - !GetAtt FixerCallbackQueue.Arn
-                  - !GetAtt FixerCallbackDeadletterQueue.Arn
```
Comment (cavis) [inline stacks/apps/feeder.yml:261]: Yeah, I guess shoryuken never directly interacts with the DLQ - it's AWS only.  +1.

**Review by cavis — APPROVED** [review]
👍 seems like that'll probably work.

File: stacks/apps/feeder.yml (hunk 1 above)
Comment (farski) [reply stacks/apps/feeder.yml:260]: https://github.com/ruby-shoryuken/shoryuken/wiki/Amazon-SQS-IAM-Policy-for-Shoryuken

It's more permissive than what Shoryuken needs. I wasn't sure if there was anything else touching the queues, given the old `sqs*` policy.

**Pull request merged**